event-stream-processing/samples/nr-broker-logs02.json

```diff
@@ -0,0 +1,49 @@
+{
+  "event": {
+    "category": "web",
+    "dataset": "generic.access",
```

Comment:
Audit one would be nice.

Reply:
Correct. The value audit would not be valid for event.category. What you're looking for is dataset: "broker.audit". The demo scripts will generate those.

https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-category.html